starter-code/index.js

```diff
@@ -1,10 +1,46 @@
-var SortedList = function() {};
+var SortedList = function(items, length) {
+    this.items = [];
+    this.length = 0;
+ };
+    
+ SortedList.prototype.add     = function(item) {
+    this.items.push(item);
+    this.items.sort((a,b)=>{
+        return a - b;
+      });
+    this.length = this.items.length;
```

Comment:
Since we are pushing into array, we should add to the length, so the proper code would be:
```
this.length++;
```